apps/src/templates/sectionAssessments/SubmissionStatusAssessmentsTable.jsx

```diff
@@ -7,13 +7,15 @@ import wrappedSortable from '../tables/wrapped_sortable';
 import orderBy from 'lodash/orderBy';
 import FontAwesome from '@cdo/apps/templates/FontAwesome';
 import color from '@cdo/apps/util/color';
+import {studentOverviewDataPropType} from './assessmentDataShapes';
 
 const TABLE_WIDTH = tableLayoutStyles.table.width;
 const TABLE_COLUMN_WIDTHS = {
   name: TABLE_WIDTH / 3,
-  numMultipleChoiceCorrect: TABLE_WIDTH / 8,
-  numMultipleChoice: TABLE_WIDTH / 8,
-  percentCorrect: TABLE_WIDTH / 8,
+  numMultipleChoiceCorrect: TABLE_WIDTH / 12,
+  numMultipleChoice: TABLE_WIDTH / 12,
+  numMatchCorrect: TABLE_WIDTH / 12,
+  numMatch: TABLE_WIDTH / 12,
   submissionTimeStamp: TABLE_WIDTH / 5
```

Comment:
This is some weird math. neither before nor after add up to 100% of TABLE_WIDTH:
```
irb(main):001:0> 1.0/3 + 3.0/8 + 1.0/5
=> 0.9083333333333332
irb(main):002:0> 1.0/3 + 4.0/12 + 1.0/5
=> 0.8666666666666667
```

Reply:
I think as long as it looks ok, and the total table width isn't weirdly different from any other tables on the page, it should be fine though?

Reply:
I updated it so it makes more sense but it still seems to be working the same 🤷‍♀